setup.py

```diff
@@ -164,7 +164,7 @@ def run(self):
 
 class Develop(develop):
     def finalize_options(self):
-        self.user = True  # always use `develop --user`
+        self.user = False  # always use `develop --user`
```

Comment:
I prefer to explicitly check conda/venv/virtualenv.
Pip will not fallback to `--user` when python directory is protected, and the error message will not suggest to do so either.

If you don't want to do the check anyway, just remove this function. Forcing to `False` will make it impossible to install as normal user without venv.